client/src/routes/send-email/+page.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+  import { onMount } from "svelte";
+  let message: string | null = null;
+  let send_email = async () => {
```

Comment:
does this need to be a separate function? maybe just replacing the function inside `onMount` with this could make it simpler? it's really subjective, so not too big of an issue

Reply:
I made it like this so that we can call it again when the `resend` button is clicked.
I don't think that you can call the function inside `onMount` again right?

Reply:
I see, that's fair